data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/Event.java

```diff
@@ -67,6 +67,14 @@ public interface Event extends Serializable {
      */
     String toJsonString();
 
+    /**
+     * Gets a serialized Json string of the specific key in the Event
+     * @param key the field to be returned
+     * @return Json string of the field
+     * @since 2.2
+     */
+    String getToJsonString(String key);
```

Comment:
This might be clearer to rename to `getAsJsonString`.

Reply:
I like the suggestion